ui-v2/app/components/hashicorp-consul.js

```diff
@@ -1,11 +1,15 @@
 import Component from '@ember/component';
 import { get, set } from '@ember/object';
-const $html = document.documentElement;
-const $body = document.body;
+import { inject as service } from '@ember/service';
 export default Component.extend({
+  dom: service('dom'),
+  // TODO: could this be dom.viewport() ?
+  win: window,
   isDropdownVisible: false,
```

Comment:
This is my first question. Now I have all these things in the `dom` service, it would make sense to get `window` from there also. I was thinking of making a `viewport` method, which would be a nice word whether window was a browser window, an iframe (not really a window) like when ember testing, or anything else. To me the viewport is what we are actually talking about here.

I'm like 90% sure I want to do this, thought it might be useful to ask what others think?